requirements/scenarios.md

```diff
@@ -310,7 +310,7 @@ It should be possible for the signer to revoke the trust in that vulnerable arti
 If a user does not have a specific key for a given artifact, verified using a third party system, they will need to determine the trusted signing key(s) for an artifact by chaining from a trusted key.
 
 1. The user determines the trusted key(s) for a specific artifact using delegations from a trusted root.
-1. The user downloads and verifies an artifact using Notary v2 and the trusted key(s) discovered in the previous step.
+1. The user downloads and verifies an artifact using Notary Project specification implementation and the trusted key(s) discovered in the previous step.
```

Comment:
The user downloads and verifies an artifact using the Notary Project specification implementation ...

Reply:
I will update it to `implementations of the Notary Project specification`